nssp/delphi_nssp/run.py

```diff
@@ -72,18 +70,28 @@ def run_module(params):
         Nested dictionary of parameters.
     """
     start_time = time.time()
-    logger = get_structured_logger(
-        __name__,
-        filename=params["common"].get("log_filename"),
-        log_exceptions=params["common"].get("log_exceptions", True),
-    )
+    custom_run = params["common"].get("custom_run", False)
+    # logger doesn't exist yet means run_module is called from normal indicator run (instead of patching)
+    if not logger:
+        logger = get_structured_logger(
+            __name__,
+            filename=params["common"].get("log_filename"),
+            log_exceptions=params["common"].get("log_exceptions", True),
+        )
+        if custom_run:
+            logger.warning("custom_run flag is on despite direct indicator run call. Normal indicator run continues.")
```

Comment:
question: is this the default behavior we want? How does this work in the other existing patchers? Maybe we should just raise an error so, e.g., the user doesn't accidentally use normal output as patch data.

Reply:
@nmdefries Existing patcher in google symptom doesn't handle this case yet. 

My thought process when I wrote this is, the accidental use of normal output as patch data won't happen because

Case 1: intention is to do normal run, just forgot to turn off flag:
Since user call the normal run (`env/bin/python -m delphi_nssp`), chances are they do want to do normal run and they just forgot to turn the custom flag off. 

Case 2: intention is to do patch run, just mistakenly trigger normal run:
If someone want to patch things but accidentally call normal run, they would find no batch issue format dirs or csv created in the patch_dir if we handle it this way. They will then check log and find their mistake. This is because output of the run under this circumstance will go into normal receiving dir instead of patch dir. 
If they stubbornly try to ingest these normal run result into our database, without the correct dir format created by correct patch run call, patch acquisition will fail anyway.

lmk if I should just error it out and halt.

Reply:
I made the option to let custom run be a possible behavior if someone want to run with custom dates that's not patching. If that's not a really a use case that needs to be considered I can also fix things on the other indicators to match this behavior

Reply:
Also the case that Nat was mentioning of running normal when wanting to do patch instead shouldn't be a concern since you have to run as `python -m delphi_indicator.patch` specifically in order to do patches and the output directory would be an indicator that you didn't run the right code